bower.json

```diff
@@ -2,7 +2,7 @@
   "name": "ember-cli-fastboot",
   "dependencies": {
     "jquery": "^1.11.1",
-    "ember": "canary",
+    "ember": "components/ember#8fababb25f6aa7c0a172bb2e3020c8fd63a8e4c1",
```

Comment:
This is locked down to the SHA just before Glimmer landing (since Fastboot support isn't ready there yet).